### DISCUSSION TITLE
Latest Nextjs 'NotFoundError'

### DISCUSSION THREAD

**Discussion by moneebalalfi** [vercel/next.js#47262] [Help]
### Summary

I was in another layout, signin/layout.tsx .. and then I clicked the back arrow .
I got this

<img width="1434" alt="Screen Shot 2023-03-18 at 00 39 48" src="https://user-images.githubusercontent.com/38296697/226058254-08772a56-78bd-41e9-b1ed-bc4e8687482e.png">

**Comment by icyJoseph** [comment]
Have you got a small reproduction demo? or some more steps to reproduce this? I tried two layouts and back and forth navigation. Using latest next as well.

For the record, this is a React error though. That's what the call stack is saying anyway.

Could you share more of that call stack?
